nbs/tests/mpl_image_tests/test_05_forest_plot.py

```diff
@@ -0,0 +1,198 @@
+import pytest
+import numpy as np
+from scipy.stats import norm
+import pandas as pd
+import matplotlib as mpl
+import os
+from pathlib import Path
+
+mpl.use("Agg")
+import matplotlib.ticker as Ticker
+import matplotlib.pyplot as plt
+
+from dabest._api import load
+
+import numpy as np
+import pandas as pd
+from scipy.stats import norm
+
+def create_delta_dataset(N=20, 
```

Comment:
Functions related to generation of test-related datasets/mock data should be under nbs/test/data directory and in its own python script.

Reply:
Thank you for the suggested changes!